css/theme-weebly.scss

```diff
@@ -908,6 +908,22 @@ div.Notice {
 	margin-right: 4px;
 }
 
+.tags .tags-link {
```

Comment:
Minor, but since these aren't actually Orbit overrides they can be moved above the Overrides section in the CSS.